src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -6830,20 +6830,23 @@ namespace HVACUnitarySystem {
 						CalcUnitarySystemToLoad( UnitarySysNum, AirLoopNum, FirstHVACIteration, CoolPLR, HeatPLR, OnOffAirFlowRatio, TempSensOutput, TempLatOutput, HXUnitOn, _, _, CompressorONFlag );
 					} else if ( CoolingLoad ) {
 						// RegulaFalsi may not find cooling PLR when the latent degradation model is used.
-						// IF iteration limit is exceeded, find tighter boundary of solution and repeat RegulaFalsi
+						// IF iteration limit is exceeded (SolFlag = -1), find tighter boundary of solution and repeat RegulaFalsi
 						TempMaxPLR = -0.1;
 						TempSysOutput = SensOutputOff;
 						TempLoad = ZoneLoad;
-						while ( ( TempSysOutput - TempLoad ) > 0.0 && TempMaxPLR < 1.0 ) {
+						while ( ( TempSysOutput - TempLoad ) > 0.0 && TempMaxPLR < 0.95 ) { // avoid PLR > 1 by limiting TempMaxPLR to 1 (i.e., TempMaxPLR += 0.1)
 							// find upper limit of HeatingPLR
 							TempMaxPLR += 0.1;
```

Comment:
My thought is clock cycles. I'd be testing for > 1 each pass through, instead of just using 0.95. I also force TempMaxPLR to 1 in the test just below since I couldn't count on += 0.1 giving me exactly 1. So the previous iteration should be TempMaxPLR = 0.9, if it needed to iterate again, it would go to 1. The alternative would be for readability and still wind up at the same place, however, can I really count on TempMaxPLR being exactly equal to 1? What if it were 0.99999999, would this cause an infinite loop? If you single step here (I did so no need) you'll see the increments are not exactly += 0.1.